nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/resources/logback-test.xml

```diff
@@ -57,11 +60,12 @@
 
     <logger name="org.apache.nifi.processors.standard" level="DEBUG"/>
     <logger name="target.file" level="DEBUG" additivity="true">
-        <appender-ref ref="TARGET_FILE" />
+        <appender-ref ref="TARGET_FILE"/>
     </logger>
 
-    <root level="INFO">
+    <!--<root level="${overrideLog:-DEBUG}">-->
+    <root level="DEBUG">
```

Comment:
Did you intend to revert this back to the override log variable and default to INFO?